src/data_types/chars.rs

```diff
@@ -7,6 +7,7 @@ use core::convert::{TryFrom, TryInto};
 use core::fmt;
 
 /// Character conversion error
+#[derive(Debug)]
```

Comment:
Do you think it would be helpful to also derive `Copy` and `Clone` while we're at it? Maybe someone will need to duplicate a `Result<_, CharConversionError>`.

Reply:
done